src/core/core.tooltip.js

```diff
@@ -787,6 +791,35 @@ module.exports = function(Chart) {
 				ctx.stroke();
 			}
 		},
+		drawVerticalGuideLine: function(vm, ctx, opacity) {
+			if (vm.verticalGuideBorderWidth <= 0) {
+				return;
+			}
+
+			var chart = this._chart;
+			var boxes = chart.boxes;
+			var body = boxes[3];
+
+			ctx.save();
+
+			ctx.strokeStyle = mergeOpacity(vm.verticalGuideBorderColor, opacity);
+			ctx.lineWidth = vm.verticalGuideBorderWidth;
+
+			var x = vm.caretX;
+
+			// for sharp solid line
+			if (ctx.lineWidth % 2 === 1) {
```

Comment:
I think you could use `helpers.aliasPixel` here.

Reply:
Oh, I see. Thank you for the guidance. I will update the code as you suggest.